thirdeye-ui/src/app/locale/languages/en-us.json

```
@@ -118,6 +118,11 @@
         "anomaly-period": "Anomaly Period",
         "anomaly-sensitivity": "Anomaly sensitivity",
         "anomaly-start": "Anomaly start",
+        "app-header": {
+            "null-mapped-name": "default(Not set)",
```

Not sure but this looks more consistent to me.

```suggestion
            "null-mapped-name": "Default(Not set)",
```

---

Done